kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -122,8 +122,10 @@ def test(args):
 
 @cli.command()
 def install():
-    """Install project dependencies from requirements.txt."""
+    """Install project dependencies from both requirements.txt and requirements.yml."""
     python_call("pip", ["install", "-U", "-r", "src/requirements.txt"])
+	os.system('conda install --file src/requirements.yml --yes')
```

Comment:
`os.system()` works but it’s not the recommended way for executing shell commands. I think it would be better if you could instead use the `call` function provided in `kedro.cli.utils`, similar to how `package()` or `build_docs()` do.

Reply:
Could you also please fix the indentation? :) so that line 126 and 127 are aligned.